led-badge-11x44.py

```diff
@@ -453,6 +454,8 @@ def header(lengths, speeds, modes, blink, ants, brightness=100):
   devinfo = pyhidapi.hid_enumerate(0x0416, 0x5020)
   #dev = pyhidapi.hid_open(0x0416, 0x5020)
 else:
+  if args.hid != "0":
+    sys.exit("HID API access is needed but not initialized. Fix your setup")
```

Comment:
What are the expected actions to "Fix your setup" ?
I guess some experts would know, but we need Usage messages for inexperience target audience pleases.

Reply:
In my case for Ubuntu 22.04 to fix it I had to:

- `sudo apt-get install -y libhidapi-libusb0`
-  ln -s /usr/lib/x86_64-linux-gnu/libhidapi-libusb.so.0 libhidapi-libusb.so.0